stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventBatchUpdate.kt

```diff
@@ -71,15 +72,15 @@ internal class EventBatchUpdate private constructor(
     }
 
     internal class Builder {
-        private val channelsToFetch = mutableSetOf<String>()
+        private val channelsToFetch = mutableSetOf<String?>()
```

Comment:
we can't fetch null channels, also we don't need to since we should store the markReadAll in 1 place.